util/reggen/uvm_reg_base.sv.tpl

```diff
@@ -503,18 +506,16 @@ ${_create_reg_field(dv_base_prefix, reg_width, reg_block_path, reg.shadowed, reg
 ##
 ##    reg              the Register that needs apply regwens
 ##
-##    reg_inst         a string giving the field of the uvm_reg_block that
-##                     should be updated. For single registers, this will just
-##                     be the register name. For elements of multi-registers,
-##                     it will be the name of an array item.
-##
-<%def name="apply_regwen(rb, reg, reg_inst)">\
```

Comment:
I don't think this is right: line 518 still uses `reg_inst`, so you'll just get some random global, which will cause a hard-to-debug error in the future (Mako's scoping rules are awful)

Reply:
I got the reg_inst in apply_regwen. But I changed it back to get the inst_name before calling apply_regwen.